yfinance/base.py

```diff
@@ -352,6 +372,13 @@ def history(self, period="1mo", interval="1d",
             df.loc[df["Stock Splits"].isna(), "Stock Splits"] = 0
         else:
             df["Stock Splits"] = 0.0
+        if is_capital_gains_data_supported:
```

Comment:
Instead, check that `capital_gains` is not None and not empty

Reply:
Sure